locales/notes.ca.properties

```diff
@@ -0,0 +1,80 @@
+# General note/notebook operations
+notebooks=Blocs de Notes
+tap-anywhere-to-start=Premeu a qualsevol lloc per iniciar
+note-info=Informacio de la nota
+add-to-contacts=Afegir a Contactes
+new-notebook=Crear Bloc de Notes
+notebook-all=Totes les notes
+notebook-trash=Paperera
+notebook-action-title=Editar Bloc de notes
+notebook-action-rename=Canviar el nom
+notebook-action-delete=Eliminar
+prompt-rename-notebook=Canviar el nom del Bloc de notes:
+prompt-delete-notebook=Premeu OK per eliminar aquest Bloc de notes. \n\nOBSERVACI&Oacute;: totes les vostres notes seran enviades a la paperera.
+note-restored=Recuperat a {{notebook}}
+new-note=Nova nota
+empty-trash=La paperera &eacute;s buida!
+first-notebook-name=El meu Bloc de notes
+empty-notebook-name=Notes
+note-cancel-changes=S'han realitzat canvis a la nota, voleu desar els canvis?
+confirm-trash-note=Premeu OK per moure aquesta nota a la Paperera
+confirm-delete-note=Esteu segurs de que voleu eliminar definitivament aquesta nota?
+add-image-title=Annexar una foto a la nota:
+image-not-supported=Aquesta caracter&iacute;stica no est&agrave; suportada en aquest terminal
+image-label=Foto
+date-updated=Data d&#39;actualitzaci&oacute;
+date-created=Data de creaci&oacute;
+title=T&iacute;tol
+notebook=Bloc de notes
+created-on=Creat
+modified-on=Modificat
+done=Fet
+search-results=Resultats de la cerca
+empty-notebook=No hi ha notes guardades<br />Comen&ccedil;ar a escriure ara
+
+# Evernote integration
+connect-with=Connectar amb
+connected-to-evernote=Connectant amb Evernote
+settings=Par&agrave;metres
```

Comment:
Ok, i have found the "delete notebook" feature using the app on my phone: holding the tap for a second on the name of the notebook at the list of notebooks. And i see that it is perfect rendered the translated message on my simulator. I put an screenshot for you here:

http://imasdeweb.com/UserFiles/Image/2015/FXOS_delete_notebook_in_catalan.png

Indeed, is it possible to upload images in this issued thread of Github? it would be more comfortable to use ;)

Cheers!

OS: i must go out the office now. Thanks!